tests/components/roborock/test_config_flow.py

```diff
@@ -316,7 +341,7 @@ async def test_discovery_already_setup(
         context={"source": config_entries.SOURCE_DHCP},
         data=DhcpServiceInfo(
             ip=NETWORK_INFO.ip,
-            macaddress=NETWORK_INFO.mac.replace(":", ""),
+            macaddress=NETWORK_INFO.mac,
```

Comment:
I think the format of this is still without ":" so its probably worth being accurate for the test input here.

Reply:
Rather: The input before was correct for how the integration does it